src/js/api/instance.js

```
@@ -1,92 +1,66 @@
-import {hyphenate, remove, within} from 'uikit-util';
+import { remove } from 'uikit-util';
+import { attachToElement, createComponent, detachFromElement, getComponent } from './component';
+import { update } from './global';
+import { callConnected, callDisconnected, callHook } from './hooks';
+import { callUpdate } from './update';
 
-export default function (UIkit) {
+export default function (App) {
+    App.prototype.$mount = function (el) {
+        const instance = this;
```

Biome

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)